Material.Blazor/Components/DatePicker/MBDatePicker.razor

```diff
@@ -73,8 +73,8 @@
                                  DateFormat="@AppliedDateFormat"
                                  DateIsSelectable="@DateIsSelectable"
                                  DateSelectionCriteria="@DateSelectionCriteria"
-                                 MaxDate="@MaxDate"
-                                 MinDate="@MinDate"
+                                 MaxDate="@CascadingDefaults.AppliedDatePickerDefaultMaxDate(MaxDate)"
```

Comment:
This line and the one below are the only non-batching changes. I recommend to make these changes again in a separate PR and abandoning this one.